orc8r/gateway/python/magma/common/cert_validity.py

```diff
@@ -90,8 +90,8 @@ def cert_is_invalid(host, port, certfile, keyfile, loop):
     tcp_res, ssl_res = res
 
     if isinstance(tcp_res, Exception):
-        logging.error(
-            'Error making TCP connection: %s, %s',
+        logging.debug(
+            'Error making TCP connection: %s, %s - thus no indication for invalid SSL certificate',
```

Comment:
```suggestion
            'Error making TCP connection: %s, %s. Thus, no indication of an invalid SSL certificate.',
```
For consistency with below.